Lesson5/src/Military.java

```diff
@@ -0,0 +1,57 @@
+public class Military extends airTransport {
+    private boolean ejectSystem;
+    private int countOfMissiles;
+
+    public Military(int power, int maxSpeed, int weight, String brand, int lengthOfRunway, float lengthOfTheWings, boolean ejectSystem, int countOfMissiles) {
+        super(power, maxSpeed, weight, brand, lengthOfRunway, lengthOfTheWings);
+        this.countOfMissiles = countOfMissiles;
+        this.ejectSystem = ejectSystem;
+    }
+
+    public boolean isEjectSystem() {
+        return ejectSystem;
+    }
+
+    public void setEjectSystem(boolean ejectSystem) {
+        this.ejectSystem = ejectSystem;
+    }
+
+    public int getCountOfMissiles() {
+        return countOfMissiles;
+    }
+
+    public void setCountOfMissiles(int countOfMissiles) {
+        this.countOfMissiles = countOfMissiles;
+    }
+
+    public void fire() {
+        if (getCountOfMissiles() != 0) {
+            System.out.println("The missile launched");
```

Comment:
I think you want to decrease missiles count here since one of them has been launched